### PR TITLE
i have changed the css for career page responsiveness

### PR DESCRIPTION
**Pull request by JyothiNandyala**
# Description

I made a few changes for css files for checking the responsive for career page .

Fixes # (issue)

# How to test?

 please open the page  and go to developer tools and check the resolution for  1024x768 to 1920x1080

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/containers/CareerPage/CareerPage.component.js [hunk 1]
```diff
@@ -70,8 +75,19 @@ const CareerPage = ({ jobTitle }) => {
...
+                label="Retake quiz"
+                size="big"
+                color="grey"
+                isMono={true}
+                onClick={handleClick}
+              />
+            </Link>
+            <Link to="/elevatorpitch">
+              <Button label="Elevator pitch" size="big" isMono={true} />
+            </Link>
```
Comment (bytewiz) [inline src/client/containers/CareerPage/CareerPage.component.js:90]: This is not the correct way to write this. think of it as it ends up being HTML and you should not wrap an <a> tag around a <button> tag.

https://stackoverflow.com/questions/6393827/can-i-nest-a-button-element-inside-an-a-using-html5

If you want to navigate to a page using a button this is the correct way to do so :).

https://reactrouter.com/web/api/Hooks/usehistory

File: src/client/containers/CareerPage/CareerPage.component.js [hunk 2]
```diff
@@ -25,8 +25,13 @@ const alternateCareerArray = [
 ];
 
 const CareerPage = ({ jobTitle }) => {
+  // onclick
+
+  const handleClick = () => {
+    console.log('clicked');
```
Comment (bytewiz) [inline src/client/containers/CareerPage/CareerPage.component.js:31]: logs are only for when developing and debugging

**Review by TITHI1244 — APPROVED** [review]
Seems moderately fine to me for the desktop versions :) Good job @JyothiNandyala

**Review by bytewiz — CHANGES_REQUESTED** [review]

**Review by dpfernandes — APPROVED** [review]
👍

**Review by bytewiz — APPROVED** [review]

**Pull request merged**